common/buildings/wc_common_buildings.txt

```diff
@@ -0,0 +1,678 @@
+﻿### aviary complex
+
+aviary_complex_01 = {
+	construction_time = quick_construction_time
+
+	can_construct_potential = {
+		building_requirement_castle_city_church = { LEVEL = 01 }
+		OR = {
+			scope:holder.culture = {
+				OR = {
+					has_innovation = innovation_taming_the_skies
+					has_innovation = innovation_flying_beasts
+			}
+			culture_has_light_aerial_maa = yes
+			culture_has_massive_aerial_maa = yes
+		}
+	}
+
+	can_construct_showing_failures_only = {
+		building_requirement_civilized = yes
+	}
+	
+	cost_gold = cheap_building_tier_1_cost
+	
+	levy = normal_building_levy_tier_1
+	
+	character_modifier = {
+		light_aerial_damage_mult = 0.03
```

Comment:
Do you have the balance spreadsheet for them?

Reply:
Yes

https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=1906407470